### PR TITLE
repair prettier for windows, add aws to project, add dependencies

### PR DESCRIPTION
**Pull request by JaCoWymowny**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,9 +18,11 @@
 .env.development.local
 .env.test.local
 .env.production.local
+/server/.env
+.env
```
Comment (kj-ninja) [inline .gitignore:22]: wtf?

File: client/.prettierrc [hunk 2]
```diff
@@ -4,5 +4,6 @@
   "printWidth": 100,
   "singleQuote": true,
   "trailingComma": "es5",
-  "bracketSameLine": false
-}
\ No newline at end of file
+  "bracketSameLine": false,
+  "endOfLine": "auto"
```
Comment (kj-ninja) [inline client/.prettierrc:8]: why's that?

File: client/package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "@types/react-dom": "^18.0.11",
     "firebase": "^9.19.1",
     "framer-motion": "^10.12.2",
+    "home-calendar": "file:..",
```
Comment (kj-ninja) [inline client/package.json:18]: wtf x100? xD

File: server/.gitignore [hunk 4]
```diff
@@ -18,9 +18,11 @@
 .env.development.local
 .env.test.local
 .env.production.local
+/server/.env
+.env
```
Comment (kj-ninja) [inline server/.gitignore:22]: ?

File: server/package.json [hunk 5]
```diff
@@ -14,7 +14,8 @@
     "axios": "^1.3.4",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "home-calendar": "file:.."
```
Comment (kj-ninja) [inline server/package.json:18]: again "home-calendar" ???

**Review by kj-ninja — CHANGES_REQUESTED** [review]

File: server/src/config/config.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import dotenv from "dotenv";
+
+dotenv.config();
+
+interface DbConfig {
+  [key: string]: any;
```
Comment (kj-ninja) [inline server/src/config/config.ts:6]: remove any

File: server/src/routes/api.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import express from 'express';
+import { Router } from 'express';
+
+const api: Router = express.Router();
+
+export default api;
```
Comment (kj-ninja) [inline server/src/routes/api.ts:6]: why you add router now?

**Review by kj-ninja — CHANGES_REQUESTED** [review]

File: client/.prettierrc (hunk 2 above)
Comment (JaCoWymowny) [reply client/.prettierrc:8]: windows thing. Line break unification.

File: server/src/models/index.ts [hunk 8]
```diff
@@ -0,0 +1,42 @@
+import fs from "fs";
+import path from "path";
+import { Sequelize, DataTypes } from "sequelize";
+import { dbConfig } from "../config/config";
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = dbConfig[env];
+
+const db = {} as any;
```
[the remark this answers is not in the document]
Comment (JaCoWymowny) [reply server/src/models/index.ts:10]: without table i dont have types

**Pull request closed without merging**